src/genie/libs/parser/ironware/tests/ShowIPOSPFInterfaceBrief/cli/equal/documentation_output_expected.py

```diff
@@ -0,0 +1,13 @@
+expected_output = {
```

Comment:
for these files, under `equal` folder, we usually have like below file name with prefix `golden_`.

```
golden_output_expected.py
golden_output2_expected.py

Reply:
@tahigash I see, are you happy to leave the files named like this? Gives a bit more description about what they're testing against. E.g. documentation_x_x is testing against output displayed in the Ironware Documentation whereas live_x_x is testing against a devices live output.

Reply:
ok. we can keep current names. having meaningful name would be more useful.